examples/README.md

```diff
@@ -53,7 +53,7 @@ before trying out the examples.
           ]
         }
   ```
-  A summary of latency in the client log is typically like the following
+  a summary of latency in the client log is typically like the following
```

Comment:
Revert?

Reply:
It's not the beginning of the sentence, or the previous sentence isn't complete.

Reply:
It is the beginning of a sentence. It is basically just missing a `:` or `.` after "following." "The following" is a noun here.

Reply:
Oh, I see. How about: "Then the latency summary in the client log is like the following"